src/views/Viewer.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="viewer">
-    <liwo-map :layerSet="activeLayerSet" />
+    <liwo-map :layer-set="activeLayerSet"/>
```

Comment:
`layer-set` should be `layerSet` imho since the variable is a JS prop and is named `layerSet` in the `liwo-map` component.